android/intltest/java/com/facebook/hermes/test/HermesIntlAndroidTest.java

```diff
@@ -17,14 +17,38 @@
 import org.junit.Test;
 
 public class HermesIntlAndroidTest extends InstrumentationTestCase {
+
+  private void evalScriptFromAsset(JSRuntime rt, String filename) {
+    try {
+      AssetManager assets = getInstrumentation().getContext().getAssets();
+      InputStream is = assets.open(filename);
+      String script = new BufferedReader(new InputStreamReader(is))
+              .lines().collect(Collectors.joining("\n"));
+      rt.evaluateJavaScript(script);
+    } catch (IOException ex) {
+      ex.printStackTrace();
+      assert (false);
```

Comment:
an exception will already cause a test failure, and allow additional tests to run and report status.  What is the reason to explicitly catch and assert here?